### PR TITLE
Move COmplete Vehicles Section to top level

### PR DESCRIPTION
**Pull request by hamishwillee**
As discussed @bkueng

### CODE REVIEW

File: en/complete_vehicles/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+This section contains information about complete vehicles that run PX4.
+
+> **Note** This is a small subset of vehicles that can run PX4.
+  Others are listed on [px4.io](https://px4.io/technology/airframes/).
+
+- Consumer drones that can be updated to run PX4 (from other flight stacks):
+  * [Crazyflie 2.0](../complete_vehicles/crazyflie2.md)
+  * [Parrot Bebop](../complete_vehicles/bebop.md)
+  * [BetaFPV Beta75X 2S Brushless Whoop](../complete_vehicles/betafpv_beta75x.md)
+  * Teal One
```
Comment (bkueng) [inline en/complete_vehicles/README.md]: This runs natively PX4

File: en/complete_vehicles/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Whole-vehicle hardware reference platforms that use PX4:
+  * [Intel® Aero Ready to Fly Drone](../complete_vehicles/intel_aero.md) - UAV development platform with high-end autopilot and integrated onboard computer.
```
Comment (bkueng) [inline en/complete_vehicles/README.md]: I'd say it's not that 'high-end' anymore

File: en/getting_started/flight_controller_selection.md [hunk 3]
```diff
@@ -20,22 +21,20 @@ Controller | Description
...
 [Qualcomm Snapdragon Flight](../flight_controller/snapdragon_flight.md) | A high-end autopilot computer that runs PX4 on the DSP (on QuRT RTOS). It includes a camera and WiFi.
-[Intel® Aero Ready to Fly Drone](../flight_controller/intel_aero.md) | A UAS development platform that integrates a powerful Linux computer, PX4 on NuttX, and a camera in a single package.
+[Intel® Aero Ready to Fly Drone](../complete_vehicles/intel_aero.md) | A UAS development platform that integrates a powerful Linux computer, PX4 on NuttX, and a camera in a single package.
 [Raspberry Pi 2/3 Navio2](../flight_controller/raspberry_pi_navio2.md) | RaPi can be connected to an autopilot and used as a companion computer.
 
 
 ## Commercial UAVs that can run PX4
 
-PX4 has been ported to a number of popular commercial drone products, as listed below. This allows you to add mission planning and other PX Flight modes to your vehicle. 
+PX4 is available on many popular commercial drone products, including some that ship with PX4 and others that can be updated with PX4 (allowing you to add mission planning and other PX Flight modes to your vehicle).
```
Comment (bkueng) [inline en/getting_started/flight_controller_selection.md]: typo: PX

**Comment by hamishwillee** [comment]
Hi @bkueng . Thanks for the review! I fixed your feedback and added new list of vehicles that support PX4 natively in https://github.com/PX4/px4_user_guide/pull/416/commits/93e9b5063b3acd1715a74a276daa1ea68115ea54#diff-708de220e4858a7cc55b77ed0e473e05R8 - can you please sanity check that.

Already merged.

**Comment by bkueng** [comment]
So I take it you did not want to make the distinction between upstream supported and PX4-based vehicles?

**Comment by bkueng** [comment]
> I thought I had - what do you mean by upstream supported? Aren't they Consumer drones that can be updated to run PX4?

No. Let me try to explain with an example: The H520 is based on PX4 (meaning Yuneec uses the PX4 Firmware/source code), and adds a separate board configuration and some other code customizations. These are not public, but they are required to run PX4 on the H520 and thus it is impossible to flash/use upstream PX4 on that vehicle.

**Comment by hamishwillee** [comment]
Thanks @bkueng . Yes it makes sense to separate those out because they are different - ie use customised PX4 from another vendor. What is the set of these? H520 - the other Yuneec's? what else?

**Comment by bkueng** [comment]
Yes it includes all of the Yuneec vehicles.
The ones from the Consumer list that are actually upstream supported are only these:
    * [Teal One](https://tealdrones.com/teal-one/)
    * [Vertical Technologies DeltaQuad](https://px4.io/portfolio/deltaquad-vtol/)

**Comment by hamishwillee** [comment]
@bkueng Just came back to this. So is this how the list would look (I found the last comment above confusing).

- Consumer drones run PX4 natively/"out of the box":
  * Multicopter
    * [AeroSense Aerobo (AS-MC02-P)](https://px4.io/portfolio/aerosense-aerobo/)
    * [Airlango Mystic](https://px4.io/portfolio/airlango-mystic/)
    * [Airdog II](https://px4.io/portfolio/airdog-ii/)
  * Fixed Wing:
    * [Sentera PXH](https://px4.io/portfolio/sentera-phx/)
  * VTOL
    * [Flightwave Edge](https://px4.io/portfolio/flywave-edge/)
    * [AeroSense VTOL](https://px4.io/portfolio/sony-aerosense/)
    * [WingtraOne Tailsitter VTOL](https://px4.io/portfolio/wingtraone-tailsitter-vtol/)
- Consumer drones that can be updated to run PX4 (from other flight stacks):
  * [Crazyflie 2.0](../complete_vehicles/crazyflie2.md)
  * [Parrot Bebop](../complete_vehicles/bebop.md)
  * [BetaFPV Beta75X 2S Brushless Whoop](../complete_vehicles/betafpv_beta75x.md)
- Whole-vehicle hardware reference platforms that use PX4:
  * [Intel® Aero Ready to Fly Drone](../complete_vehicles/intel_aero.md) - UAV development platform with autopilot and integrated onboard computer.
- Consumer drones run a custom version of PX4 (supported by their vendors):
  * Multicopter
    * [Teal One](https://tealdrones.com/teal-one/)
    * [Yuneec Typhoon H Plus](https://px4.io/portfolio/yuneec-typhoon-h-plus/)
    * [Yuneec Mantis Q](https://px4.io/portfolio/yuneec-mantis-q/)
    * [Yuneec H520](https://px4.io/portfolio/yuneec-h520-hexacopter/)
  * VTOL
    * [Vertical Technologies DeltaQuad](https://px4.io/portfolio/deltaquad-vtol/)

**Comment by Antiheavy** [comment]
@hamishwillee.  Sentera PHX fixed wing falls under the “custom version of px4” category.  Thanks.

**Comment by bkueng** [comment]
@hamishwillee this is how it should be (some of them I'm not quite sure, most likely the fall into custom):
- Consumer drones run PX4 natively/"out of the box":
  * Multicopter
    * [Teal One](https://tealdrones.com/teal-one/)
  * VTOL
    * [Vertical Technologies DeltaQuad](https://px4.io/portfolio/deltaquad-vtol/)
- Consumer drones that can be updated to run PX4 (from other flight stacks):
  * [Crazyflie 2.0](../complete_vehicles/crazyflie2.md)
  * [Parrot Bebop](../complete_vehicles/bebop.md)
  * [BetaFPV Beta75X 2S Brushless Whoop](../complete_vehicles/betafpv_beta75x.md)
- Whole-vehicle hardware reference platforms that use PX4:
  * [Intel® Aero Ready to Fly Drone](../complete_vehicles/intel_aero.md) - UAV development platform with autopilot and integrated onboard computer.
- Consumer drones run a custom version of PX4 (supported by their vendors):
  * Multicopter
    * [Yuneec Typhoon H Plus](https://px4.io/portfolio/yuneec-typhoon-h-plus/)
    * [Yuneec Mantis Q](https://px4.io/portfolio/yuneec-mantis-q/)
    * [Yuneec H520](https://px4.io/portfolio/yuneec-h520-hexacopter/)
    * [Airlango Mystic](https://px4.io/portfolio/airlango-mystic/)
    * [Airdog II](https://px4.io/portfolio/airdog-ii/)
    * [AeroSense Aerobo (AS-MC02-P)](https://px4.io/portfolio/aerosense-aerobo/) *I don't know*
  * Fixed Wing:
    * [Sentera PXH](https://px4.io/portfolio/sentera-phx/)
  * VTOL
    * [Flightwave Edge](https://px4.io/portfolio/flywave-edge/)  *I don't know*
    * [AeroSense VTOL](https://px4.io/portfolio/sony-aerosense/)  *I don't know, and the link is dead*
    * [WingtraOne Tailsitter VTOL](https://px4.io/portfolio/wingtraone-tailsitter-vtol/)

**Comment by hamishwillee** [comment]
Thanks @bkueng (and @Antiheavy ). I've updated as indicated. Removed the item with broken link - couldn't find a good replacement and there isn't much "realness" about it. I left the "I don't know" ones in place - better for users not to expect they can update them directly than to expect they can and not be able to.

**Pull request merged**